### ISSUE TITLE
tabs won't get same background color

### ISSUE THREAD

**Issue by novia713** [trufae/Therm#51]
tabs appear always in black/gray color, besides the background color
see the following image
![captura de pantalla 2018-06-20 a las 11 33 03](https://user-images.githubusercontent.com/423279/41650237-d64d2ae4-747d-11e8-8ff6-25dd95f3bb55.png)

**Comment by trufae** [comment]
i put tabs in the bottom, which is the default setting, and it should be maybe enforced always like this, so it doesnt looks as crappy as you see in this screenshot. it is fine for you to remove the tab position option from settings and enforce them at the bottom?

**Comment by novia713** [comment]
yes, i can move tabs to the bottom, but that doesn't resolve the problem :-/

**Comment by trufae** [comment]
Agree. Will see what can i do when i have some time

Thanks
